README.md

```diff
@@ -21,3 +21,8 @@ $ go build ./cmd/...
 **Notes:**
 
 - `kf` CLI must be built outside of the `$GOPATH` folder unless you explicitly use `export GO111MODULE=on`.
+
+
+## Releasing
```

Comment:
We should rename this something like "Building kf using ko"

The install instructions walk through the more standard install without using `ko`. We should probably recommend contributors use `ko` for development purposes, but I'd prefer we offer releases and install instructions that don't require the user to be setup to compile go.

Reply:
I added some additional context. I think we're going to use ko for development and to release official files you can kubectl apply to install it; that file will reference any official images we produce.

I'm still not sure about building binary artifacts for release (the kf command) but that could easily be done in almost any build pipeline.